File: dongyimai_sellergoods_interface/src/main/java/com/offcn/sellergoods/service/BrandService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    public PageResult findPage(TbBrand brand,int pageNum,int pageSize);

   public void add(TbBrand brand);

   public void delete(Long[] ids);
   public void update(TbBrand brand);
   public TbBrand findOne(Long id);
   public List<Map> selectOptionList();
}

File: dongyimai_sellergoods_web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {

    this.findByPage= function(searchEntity,pageNum,pageSize){
        return $http.post("../brand/findPage.do?pageNum=" + pageNum + "&pageSize=" + pageSize, searchEntity);
    }

    this.add = function(entity){
        return $http.post("../brand/add.do", entity);
    }

    this.update = function(entity){
        return $http.post("../brand/update.do",entity);
    }

    this.findOne = function(id){
        return $http.get("../brand/findOne.do?id=" + id);
    }

    this.delete = function(ids){
        return $http.get("../brand/delete.do?ids=" + ids)
    }

    this.selectOptions = function(){
        return $http.get("../brand/selectOptions.do");
    }


    this.selectOptionsList=function () {
        return $http.get("../brand/selectOptionList.do");
    }





})